### DISCUSSION TITLE
Refine Rumination #10

### DISCUSSION THREAD

**Discussion by busstoptaktik** [busstoptaktik/geodesy#116] [General]
[Rumination #10](https://github.com/busstoptaktik/geodesy/blob/main/ruminations/010-rumination.md) "What's wrong with ISO-19111", is long, convoluted and covers a highly specialized subject. Nevertheless, it is a core element of the [Geodesy](https://github.com/busstoptaktik/geodesy/) objective of contributing to the **development of better, simpler, and more tractable, geospatial standards, transformations, and software.**

Hence, it is important for this text to be as clear and accessible as possible, and any input towards this goal is highly appreciated. Knowing that the barrier-of-entry for a pull request may limit the amount of input, this discussion entry is intended as a quick and easy way to comment, discuss, and enhance the rumination.

(A huge thank you to @heidivanparys and @desruisseaux who have contributed immensely already with comments and opinions)

**Comment by desruisseaux** [comment]
In the _"described in 19157 (WKT)"_ sentence, do you mean _"described in 19162 (WKT)"_? The former ISO standard is for Data Quality.

Regarding ISO 19107 `DirectPosition`, maybe a little bit of historical context would help. When ISO 19111:2019 was written, the ISO 19107 revision (also published in 2019) was not yet completed. We had some insight about what was going on and indeed some ISO 19111 parts anticipated upcoming 19107 changes, but not all of them. In particular, the "coordinate system" stuff of ISO 19107 was still debated at that time. In the bibliography (annex H of OGC document), ISO 19111 explicitly refers to ISO 19107:2003, not 19107:2019. So I think that the use of `DirectPosition` in ISO 19111 should be understood as _"direct position as it was defined in 2003"_, which did not have the problem described in this rumination document. However, the problem described here would indeed need to be addressed if a future ISO 19111 revision wants to upgrade its dependency from ISO 19107:2003 to 19107:2019.

**Reply by busstoptaktik** [reply]
> In the "described in 19157 (WKT)" sentence, do you mean "described in 19162 (WKT)"? The former ISO standard is for Data Quality

Thanks, Martin @desruisseaux for pointing out this blunder once again, and sorry for missing correcting it already first time!

Also, that's some very interesting remarks regarding 19107. I will take a look in the 2003 version. I consider adding a "Comments" appendix to the text, where this kind of material can be collected for reference (obviously with proper attribution). Does this sound OK to you?

**Reply by desruisseaux** [reply]
No problem of ISO document number. An annex for historical comments looks fine to me.
